packages/stark-ui/src/modules/action-bar/components/action-bar.component.html

```diff
@@ -4,10 +4,10 @@
 		<div *ngFor="let action of actionBarConfig.actions; trackBy: trackAction" [id]="actionBarId + '-' + action.id"
 		(click)="onClick(action, $event)" class="stark-action-bar--action">
 			<ng-container *ngIf="action.isVisible !== false">
-				<button [color]="action.buttonColor ? action.buttonColor:'primary'" [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label" [disabled]="!action.isEnabled" mat-icon-button>
+				<button [color]="action.buttonColor ? action.buttonColor:buttonColor" [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label" [disabled]="!action.isEnabled" mat-icon-button>
```

Comment:
Could you please add a space before and after the `:` for having `(action.buttonColor ? action.buttonColor : buttonColor)` ?
(Prettier should do it automatically normally 😕)

Reply:
Well, in fact Prettier doesn't support HTML yet 😩
https://prettier.io/docs/en/language-support.html
https://github.com/prettier/prettier/issues/1882